api/src/routes/edit.test.ts

```diff
@@ -58,15 +62,17 @@ describe("edit", () => {
             jest.spyOn(DatastreamManager, "getInstance").mockReturnValue(datastreamManager);
         });
 
-        it("will download the datastream content", async () => {
-            datastreamManager.getMimeType.mockResolvedValue("test1");
-            datastreamManager.downloadBuffer.mockResolvedValue("test2");
+        xit("will download the datastream content", async () => {
+            datastreamManager.getMimeType.mockResolvedValue(fileType);
+            datastreamManager.downloadBuffer.mockResolvedValue(buffer);
 
-            await request(app)
+            const response = await request(app)
                 .get(`/edit/object/${pid}/datastream/${datastream}/download`)
                 .set("Authorization", "Bearer test")
                 .expect(StatusCodes.OK);
 
+            //  expect(response.headers["Content-Type"]).toEqual(fileType);
```

Comment:
Is there a reason this is commented out?

Reply:
Made chnage.